spec/lib/serde_spec.rb

```diff
@@ -37,6 +37,31 @@
         }
       )
     end
+
+    context 'when language fields contain HTML/CSS' do
+      let(:html) do
+        <<~HTML
+          <div style="height: 100px; width: 100px;"></div>
+          <p>hello & goodbye!</p>
+        HTML
+      end
+
+      let(:original_hash) do
+        ActiveSupport::HashWithIndifferentAccess.new(
+          title: '<b><a href="http://foo.com/">foo</a></b><img src="bar.jpg">',
+          description: html,
+          content: "this <b>is</b> <a href='http://gov.gov/url.html'>html</a>"
+        )
+      end
+
+      it 'sanitizes the language fields' do
+        expect(serialize_hash).to match(hash_including(
+          title_en: 'foo',
+          description_en: 'hello & goodbye!',
+          content_en: 'this is html'
+        ))
+      end
+    end
```

Comment:
I moved these specs out of the document specs:
https://github.com/GSA/i14y/pull/132/files#diff-ace14829a1667febf42459b631d07157bddc6e47d372750a17d6147b16221ceeL43

The functionality is implemented in the `Serde` module (the Document serializer), so it makes sense to move the unit specs there, and it allows me to remove the Document specs based on the soon-to-be-removed `Document.create` method.